examples/dao/assets/dao-app-approval.py

```diff
@@ -136,11 +136,13 @@ def scratch_proposal_active(idx: Int):
     max_duration = Bytes("max_duration")
     # a url with more information about the DAO
     url = Bytes("url")
+     # dao name
+    dao_name = Bytes("dao_name")
```

Comment:
you're adding more storage to global state in contract. But you're not updating the global state schema during deployment.

Reply:
I checked it. Number global state still is 2 slots.

Reply:
`globalBytes: 2` yes we have 2 slots